x-pack/plugins/code/public/components/main/main.scss

```diff
@@ -19,3 +19,106 @@
 .code-auto-margin {
   margin: auto;
 }
+
+.code-navigation__sidebar {
```

Comment:
I'd try to follow the naming conventions of the rest of Kibana / EUI. This going for everything in this page.

`.codeNavigation__sidebar`

Reply:
I like that better. I was becoming overwhelmed by hyphens. :)

Reply:
Yep, there's some docs at the bottom of the sass guidelines but generally the following is the formula.

```
.appComponent
.appComponent__element
.appComponent--modifier
.appComponent__anotherElement--modifier
.appComponent-isState
```